### PR TITLE
Issue 841 - Adding right-click menu file highlight.

### PR DESCRIPTION
**Pull request by Th30**
Hello,

I managed to fix the highlighting in the file-tree when right clicking to change to the theme color as seen below:
screen shot 2017-02-07 at 7 29 35 pm

I've also tried out orange:

screen shot 2017-02-07 at 7 37 49 pm

But decided to stick to the Mozilla Thimble colors. 

I did notice a quick bug if I kept the opacity of the "filetree-context-extension" class to anything visible when creating a folder as seen here:

screen shot 2017-02-07 at 7 32 11 pm

The problem is only present when creating the folder for the first time, after which it would go back to normal:
screen shot 2017-02-07 at 7 20 44 pm

Let me know how this can be improved or if it needs changing:) 

Regards,

Th30

### CODE REVIEW

**Comment by humphd** [comment]
This bug has been much hated by all of us for literally years!  It's awesome to see you diving into it.  I'm going to defer to @flukeout on the actual styling we want to use when we right-click.  But the fact that you're starting to figure out *where* these changes need to go, and *what* kinds of changes make sense is amazing.  Great work @Th30!

**Comment by gideonthomas** [comment]
Fixes mozilla/thimble.mozilla.org#841

**Comment by flukeout** [comment]
Checking this out now @Th30

**Comment by flukeout** [comment]
Okay @Th30 - had a look into this and here are my recommendations.

There are actually two things happening when you right-click a file.

1. A ".context" class is added to the menu item
2. A ".filetree-context" div is placed overtop of that item as the highlight

That's why the text color of the selected item isn't changing even though you have a ``color: white;`` as a rule.

Here's what I'd like to see:

1. For the selected item, change the text color to ``rgba(255,255,255,.6)``
2. For the ``filetree-context`` highlight element..
  * Change the background to ``rgba(255,255,255,.06)``
  * Make the height of the highlight match the height of the highlight for the currently opened file, which is 27px

Let me know if you have questions or need guidance.

**Comment by Th30** [comment]
OK I'll get right on it, thank you for the help!

**Comment by Th30** [comment]
Hello,

So I think I've made some progress. Just wanted to show a quick image to make sure that it's to standard:)
screen shot 2017-02-09 at 11 23 59 pm

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css [hunk 1]
```diff
@@ -43,10 +43,21 @@ li.jstree-leaf>a>span.extension {
 	width: 100% !important;
 }
 .filetree-selection-extension {
-	background-color: #333333;
-	border: none;
+    opacity: 0.0;
+}
+.filetree-context {
+    background-color: rgba(255,255,255,0.6);
```
Comment (flukeout) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: .06 For this one please.

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css [hunk 2]
```diff
@@ -43,10 +43,21 @@ li.jstree-leaf>a>span.extension {
 	width: 100% !important;
 }
 .filetree-selection-extension {
-	background-color: #333333;
-	border: none;
+    opacity: 0.0;
```
Comment (flukeout) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: Please change to just ``opacity: 0;``, we don't need the extra decimal.

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css (hunk 1 above)
Comment (Th30) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: Done:)

**Comment by flukeout** [comment]
Hey awesome @Th30 - I swear this the last thing and thank you for your patience, can you make sure that the indentation of your code matches the rest of the file's conventions? I believe that file is using a tab character while you're using spaces. Thank you!

**Comment by Th30** [comment]
@flukeout 

HAHA what are you sorry for? This is the coders life! Sure I'll make sure it's all indented asap:) Expect to see a push soon!

**Comment by flukeout** [comment]
WE DID IT. Merged - thanks so much @Th30 !!

**Comment by humphd** [comment]
Love this.  @Th30, there are a bunch of related bugs for this that you might decide to tackle next.  I don't have them all handy, but we should look through our open bugs here and in Brackets and get you to fix the rest next.  Nice work.

**Comment by Th30** [comment]
@humphd 
@flukeout 
I was actually thinking of browsing through bugs and just trying to go outside my comfort zone a little now that I'm more familiar with git, vagrant and npm environments. 

Looking forward to working with everyone to keep squashing! If any come to mind specifically or that should be next shoot me a message other wise I'll look around! :)

**Pull request merged**